FoodFriends/app/src/main/java/com/example/foodfriends/activities/MainActivity.java

```diff
@@ -97,33 +99,91 @@ public void onClick(View v) {
             @Override
```

Comment:
Hmmm could we look into a tab layout with a viewpager? It might automatically handle these cases automatically.

Reply:
I think using a View Pager would be good as it would also allow the user to swipe between tabs. However, I think I might include it as a stretch feature. I did some research and there seems to be a lot of overhead involved, and since current method works, don't want to spend too much time on it for the first deliverable.

Reply:
Integrate ViewPager into Tab Layout
Resources:
 - https://www.journaldev.com/12958/android-tablayout-viewpager
 - https://developer.android.com/guide/navigation/navigation-swipe-view